.github/ISSUE_TEMPLATE/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
+## This Community doc request is a:
+- [ ] New topic request
+- [ ] Topic content updates
+- [ ] Other
+
+<!-- (REQUIRED) What is the Community issue and code PR? -->
+## Community GitHub issue link:
+
+## Community GitHub code PR link:
+
+<!-- (REQUIRED) What is the Magento Open Source/Commerce and version for the PR? Will it be back or forward ported? -->
+## Magento version:
```

Comment:
Is this really necessary? Aren't all contributions submitted to Open Source and then ported to Commerce by internal staff later, if at all?

Reply:
Will verify.